htsget_server/drs_openapi.yaml

```diff
@@ -476,45 +513,9 @@ components:
             = md5( concat( 5e089d29, 72794b6d ) )
             = md5( 5e089d2972794b6d )
             = f7a29a04
-        contents:
-          type: array
-          description: The specific genomic contents objects that comprise this genomic DRS entity. Should contain a GenomicDataContentsObject, an optional GenomicIndexContentsObject, and SampleContentsObjects corresponding to the samples analyzed in the genomic data object.
-          minItems: 1
-          items:
-            anyOf:
-              - $ref: '#/components/schemas/GenomicDataContentsObject'
-              - $ref: '#/components/schemas/GenomicIndexContentsObject'
-              - $ref: '#/components/schemas/SampleContentsObject'
-        description:
-          type: string
-          description: type of sequencing data
-          enum:
-            - wgs
-            - wts
-        program:
-          type: string
-          description: The program this object was ingested as part of
-        reference_genome:
-          type: string
-          description: the reference genome used to compile this genomic object
-          enum:
-            - hg37
-            - hg38
-        indexed:
-          type: integer
-          description: 1 if the object has been indexed for search, 0 if not
-        aliases:
-          type: array
-          items:
-            type: string
-          description: >-
-            A list of strings that can be used to find other metadata
-            about this `DrsObject` from external metadata sources. These
-            aliases can be used to represent secondary
-            accession numbers or external GUIDs.
-    GenomicDataDrsObject:
+    AnalysisDataDrsObject:
       type: object
-      description: A DrsObject that describes a genomic data file, e.g. a variant or read file.
+      description: A DrsObject that describes a analysis data file, e.g. a variant or read file.
```

Comment:
again, no reads (read = fastq)?

Reply:
In the htsget spec, reads are bam/cram. https://samtools.github.io/hts-specs/htsget.html

Reply:
I was correct to stop leaving comments and check my assumptions. Thanks for the clarification!